CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # scikit-bio changelog
 
+## Version 0.5.8
+
+### Features
+
+### Backward-incompatible changes [stable]
+
+### Backward-incompatible changes [experimental]
+
+### Performance enhancements
+
+### Bug fixes
+
+### Deprecated functionality [stable]
+
+### Deprecated functionality [experimental]
+
+### Miscellaneous
+
+* Update hdmedians package to a version which doesn't require an initial manual numpy install.
```

Comment:
Would you mind including this in the 0.5.7 section? At some point we mixed the 0.5.6 notes with the 0.5.7 and now everything is scrambled. If you can put this in 0.5.7 I'll submit a later PR and clean this up.

Reply:
I hope 8a72c7a covers this - please ping me if I've got myself muddled :-)